activerecord/lib/active_record/associations/collection_proxy.rb

```diff
@@ -1149,6 +1148,18 @@ def find_from_target?
         def exec_queries
           load_target
         end
+
+        def respond_to_missing?(method, _)
+          scope.respond_to?(method) || super
```

Comment:
@matthewd As you pointed out, I fixed to define `respond_to_missing?` instead of `respond_to?`. Thanks!